content/doc/developer/testing/index.adoc

```diff
@@ -148,6 +148,36 @@ Most Java IDEs should be able to run JUnit tests and report on the results.
 //==== From the Command Line
 //==== From an IDE
 
+=== Performance considerations
+
+The test runner (Surefire) supports running tests in parallel to speed them up.
+A possibility to configure this machine-wide is to adjust the `forkCount` within Maven.
+The default setting is `forkCount=1`, which means no parallel testing.
+An often-used setting is `1C`, which spawns as many testing processes in parallel as CPU cores are present.
+If you have a machine with many cores it might be faster to set it to a smaller number, like `0.45C`.
```

Comment:
```suggestion
If you have a machine with many cores, it might be faster to set it to a smaller number, like `0.45C`.
```
small adjustment for punctuation